### PR TITLE
[All hosts](SSO) updating links and code for moving SSO samples

### PR DESCRIPTION
**Pull request by davidchesnut**
We moved the SSO samples to the PnP repo. This PR is to update links from docs to the correct new locations. Also includes any code fixes to docs to match any code fixes to the samples.

### CODE REVIEW

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
The link changes look good to me. I'll leave it to Ricky to verify the code updates.

File: docs/develop/create-sso-office-add-ins-aspnet.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ If you chose "Accounts in this organizational directory only" for **SUPPORTED AC
     var retryGetAccessToken = 0;
 
     async function getGraphData() {
-        await getDataWithToken({ allowSignInPrompt: true, allowConsentPrompt: true, forMSGraphAccess: true });
+        await getDataWithToken();
```
Comment (Rick-Kirkham) [inline docs/develop/create-sso-office-add-ins-aspnet.md:175]: Why is the parameter being removed? It's still in the sample: line 23 of the file: https://github.com/OfficeDev/PnP-OfficeAddins/blob/b6ce411d2ed935df4e68ec2185a9164fd64bef77/Samples/auth/Office-Add-in-ASPNET-SSO/Complete/Office-Add-in-ASPNET-SSO-WebAPI/Scripts/HomeES6.js

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]
Just one code change that doesn't look right.

File: docs/develop/create-sso-office-add-ins-aspnet.md (hunk 1 above)
Comment (davidchesnut) [reply docs/develop/create-sso-office-add-ins-aspnet.md:175]: For the flow of the start and the order of explanation of these parameters, this works better. I realize the complete code is slightly different.

Comment (Rick-Kirkham) [reply docs/develop/create-sso-office-add-ins-aspnet.md:175]: If the options aren't passed in at this point, then they don't get passed to getAccessToken. I think we have to keep this parameter, but add a bullet just above the snippet saying that the parameter will be explained in a later step.

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]
Still have a problem with that parameter.

**Comment by AlexJerabek** [comment]
Hi @davidchesnut,
Just pinging this thread to see what the status is.

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**